app/models/course/assessment/question.rb

```diff
@@ -14,6 +14,26 @@ class Course::Assessment::Question < ApplicationRecord
   delegate :to_partial_path, to: :actable
   delegate :question_type, to: :actable
 
+  # Experience points associated with this question across all the assessments it belongs to.
+  # Used to calculate the total number of experience points associated with a Skill.
+  #
+  # @return [Integer] The total amount of exp the question is worth.
+  def experience_points_from_all_assessments
+    question_assessments.map(&:assessment).reduce(0) do |exp, assessment|
```

Comment:
This is right now but conceptually wrong because you might duplicate questions across assessments from different courses and in those cases we don’t want to be including those in our calculation?

Reply:
Added a `course` parameter and filtered the assessments by `course`.